BoardConfig.mk

```diff
@@ -28,6 +28,7 @@ endif
 PRODUCT_PLATFORM := seine
 
 BOARD_KERNEL_CMDLINE += androidboot.hardware=pdx201
+BOARD_KERNEL_CMDLINE += androidboot.fstab_suffix=pdx201
```

Comment:
There is no need to have second line adding value to already defined variable. Please just use the one above and just add space at the end and add the fstab suffix :)

Reply:
Ofc do this for the 2 PRs on pdx20[3/6] too

Reply:
> There is no need to have second line adding value to already defined variable. Please just use the one above and just add space at the end and add the fstab suffix :)

Thanks, I forgot it, going to fix it. :P

Reply:
No, keep it on two lines. It's more readable.

Reply:
@ix5 read the commit she did. Its readable even now.

Reply:
And i mean come on we test stuff before we send it. And even if this is why we do reviews :)

Reply:
Let an old guy tell you it _does_ happen. https://github.com/sonyxperiadev/device-sony-tone/pull/170/files

Reply:
More often than you'd think https://github.com/sonyxperiadev/device-sony-common/commit/1f76f2cdd2ab24aedaf4917760293e43f5c99161

Reply:
> And even if this is why we do reviews

We've had our fair share of broken ninja-merges :wink:

Reply:
Again we are smarter now and review code before merging it

Reply:
> Again we are smarter now and review code before merging it

Do we now? https://github.com/sonyxperiadev/device-sony-seine/commit/51924408895e2cc1cbe62248e2ecf047297012cc

Reply:
Come on man. 21st sep.